pcsx2/ImGui/FullscreenUI.cpp

```diff
@@ -999,16 +998,6 @@ void FullscreenUI::DoReset()
 	});
 }
 
```

Comment:
DoChangeDisc() should be removed too, and the call site replaced with DoDiscChangeFromFile().